src/WebJobs.Script/Metrics/HostMetrics.cs

```diff
@@ -16,13 +16,18 @@ public class HostMetrics : IHostMetrics
         private readonly ILogger _logger;
 
         public const string MeterName = "Microsoft.Azure.WebJobs.Script.Host.Internal";
+        public const string FaasMeterName = "Microsoft.Azure.Functions.Host";
```

Comment:
Curious why this isn't `Microsoft.Azure.WebJobs.Script.Host`

Reply:
Rohit and I discussed this offline - we wanted a more intuitive name for the customer. While it is true the meter name normally is the assembly name, but I think we don't have to follow that as in out of proc this meter/assembly is never loaded along side customer code.

Reply:
That's fair, thanks!